### DISCUSSION TITLE
[Feature] add search option for people on mobile

### DISCUSSION THREAD

**Discussion by SomeThink1729** [immich-app/immich#13895] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

At the search page when selecting people, there is no dropdown (like at Places and cameras ...) or search. 
I'd like to have a search feature since it's hard to find a person when you have many different people and don't know how often the person appears.
the search option could be introduced right below the text and above the top person

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
Do you mean this option? 
![image](https://github.com/user-attachments/assets/12d7ca00-067a-40b3-b061-d4b6aa3ab061)

**Reply by SomeThink1729** [reply]
yeah but on mobile, sorry I saw that it wasn't clear on first look so i added it to the title. I know the option in web, but I'd like to have it on mobile as well

**Reply by alextran1502** [reply]
~~You can also search for people on the mobile app, it is the first filter option on the search page~~ nvm I see what you mean

**Comment by bo0tzz** [comment]
#11749
